src/main/java/com/gdschongik/gdsc/domain/member/domain/Requirement.java

```diff
@@ -43,4 +43,8 @@ public static Requirement createRequirement() {
     public boolean isUnivPending() {
         return this.univStatus == PENDING;
     }
+
+    public void updateUnivState(RequirementStatus univStatus) {
```

Comment:
변수명이랑 통일감 있게 가면 좋을 것 같습니다.
```suggestion
    public void updateUnivStatus(RequirementStatus univStatus) {
```